### PR TITLE
Add provider class for SolutionInfo and file which point at it

### PR DESCRIPTION
**Pull request by jarek170989**
@lauraluiz I'm not sure what values we want to put as "name", "website" or "emergencyContact". I guess it should be hardcoded,right?

### CODE REVIEW

**Comment by lauraluiz** [comment]
What happens if you don't define them? I would leave it empty and then in Sunrise Starter Project we would make it easier to add the missing info.

**Comment by jarek170989** [comment]
There won't be any errors. The user agent in corresponding places will have blank text: "commercetools-jvm-sdk/1.4.1 (AHC/2.0) Java/1.8.0_92-b14 (Mac OS X; x86_64) SOLUTION_NAME/SOLUTION_VERSION (+https://website.tld; +info@SOLUTION.com)".  So i'll change it to be empty.

**Comment by jarek170989** [comment]
@lauraluiz I've updated pr.

File: common/app/com/commercetools/sunrise/common/SunriseSolutionInfo.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.commercetools.sunrise.common;
+
+import io.sphere.sdk.client.SolutionInfo;
+import io.sphere.sdk.meta.BuildInfo;
+
+public class SunriseSolutionInfo extends SolutionInfo {
+    public SunriseSolutionInfo() {
+        setName("");
+        setVersion(BuildInfo.version());
```
Comment (lauraluiz) [inline common/app/com/commercetools/sunrise/common/SunriseSolutionInfo.java]: Have you checked this yields the correct build info for Sunrise? This seems to be coming from the JVM SDK.

File: common/app/com/commercetools/sunrise/common/SunriseSolutionInfo.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package com.commercetools.sunrise.common;
+
+import io.sphere.sdk.client.SolutionInfo;
+import io.sphere.sdk.meta.BuildInfo;
+
+public class SunriseSolutionInfo extends SolutionInfo {
+    public SunriseSolutionInfo() {
+        setName("");
```
Comment (lauraluiz) [inline common/app/com/commercetools/sunrise/common/SunriseSolutionInfo.java]: Shouldn't it have `commercetools-sunrise-java`?

File: common/app/com/commercetools/sunrise/common/SunriseSolutionInfo.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+        setVersion(BuildInfo.version());
+        setWebsite("");
+        setEmergencyContact("");
```
Comment (lauraluiz) [inline common/app/com/commercetools/sunrise/common/SunriseSolutionInfo.java]: Do we need to explicitly set the website and contact as empty strings? If possible, let's avoid calling the setters if we are not going to set anything.

**Comment by lauraluiz** [comment]
Duplicated https://github.com/commercetools/commercetools-sunrise-java/pull/614

**Pull request closed without merging**